core/src/main/java/org/mskcc/cbio/portal/scripts/ImportProfileData.java

```diff
@@ -87,6 +87,13 @@ public void run() {
             } else if (geneticProfile.getGeneticAlterationType() == GeneticAlterationType.FUSION) {
                 ImportFusionData importer = new ImportFusionData(dataFile, geneticProfile.getGeneticProfileId(), genePanel);
                 importer.importData();
+            } else if (geneticProfile.getGeneticAlterationType() == GeneticAlterationType.TREATMENT_RESPONSE) {
+
+                // add all missing Treatments to the database
+                ImportTreatments.importData(dataFile);
```

Comment:
The close coupling of the treatment meta data and treatment response data is a design feature. Treatments are very different from real `genetic entities` in the sense that there is no central point of truth of what are valid treatments. We envisioned that treatments may be different between studies. We did not want to have to repopulate the database when a new study is added that contains (some) different treatments. Instead, the treatment identifiers + meta data is loaded from the data file at moment of import. So yes, in the current implementation the importing of the meta data is necessarily tied to the import of the response data because treatments are too fluid to be part of any seed database.